shared/common-adapters/custom-emoji.native.tsx

```diff
@@ -7,7 +7,7 @@ import {Box2} from './box'
 const CustomEmoji = (props: Props) => {
   const {size, src} = props
   return (
-    <Box2 direction="vertical" style={styles.container}>
+    <Box2 direction="vertical" style={Styles.collapseStyles([{height: size}, styles.container])}>
```

Comment:
Oh whoops I think I told you the wrong thing, looks like you need `height` and `width` set on the box to stop android crashes.